lib/ctx.cpp

```diff
@@ -451,9 +451,11 @@ grn_ctx_impl_init(grn_ctx *ctx)
                       ctx,
                       grn_msgpack_buffer_write);
 #endif
-  ctx->impl->output.arrow_stream_writer = NULL;
   grn_timeval_now(ctx, &ctx->impl->tv);
 #ifdef GRN_WITH_APACHE_ARROW
+  ctx->impl->output.arrow_stream_writer = NULL;
+  ctx->impl->output.arrow_metadata_data_type = NULL;
+  GRN_RAW_STRING_INIT(ctx->impl->output.arrow_metadata_label);
```

Comment:
We should close all `ctx->impl->output.*` initialization.

Reply:
fix: [6b3174e](https://github.com/groonga/groonga/pull/2399/commits/6b3174e407fbddde0cdd34deb4313e5b8a340942) Sure.